boilerplate/openshift/golang-osd-operator-osde2e/README.md

```diff
@@ -30,11 +29,10 @@ following:
 
 | `make` target      | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
 |--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `e2e-harness-generate` | Generate scaffolding for an end to end test harness. The `test/e2e` directory is created where the tests and test runner reside. The harness has access to cloud client and harness passthrough secrets within the test job cluster. Add your operator  related ginkgo e2e tests under the `test/e2e/<operator-name>_tests.go` file. See [this README](https://github.com/openshift/osde2e-example-test-harness/blob/main/README.md#locally-running-this-example) for more details on test harness. |
```

Comment:
Did something change about the testing, that we are removing this step to generate harness?

Reply:
Nvm, just saw that the info moved to a different lovation.